### DISCUSSION TITLE
6.1.3 Linux stops ingesting

### DISCUSSION THREAD

**Discussion by cfc62** [chuot/rdio-scanner#95] [General]
Not a lot of info here, sorry about that. I started another thread on password issues, I decided to give a new Linux box a whirl, and it was great for about 30 minutes, then it stopped ingesting.

There was no error thrown that I could see, and ingesting stopped roughly 10 minutes before there was a "pruning database" message in the logs. Recordings continued to build in the recordings directory for another 10 minutes before I killed the rio-scanner process. Restarting was successful (or has been so far).

No other messages seen. The rdio-scanner process was at 0% cpu when I killed it. 

SDRTrunk also on this same box
Linux (Ubuntu 20.04), all SSD storage, 6 cores 12 threads, 32G RAM
Rdio-scanner 6.1.3

**Comment by cfc62** [comment]
Ah, should have made this an issue, I will when I get a moment.

**Comment by mjmikolay** [comment]
I had the same issue where the database just started pruning (I have it set for 14 days and today is the day I've switched to 6.1.3 but I elected to go with a clean / fresh start so the database was essentially empty. 

Restart is now throwing this error about 3-5 seconds after the program starts:
2022/01/20 14:27:16 client.processmessage: controller.processmessage.commandlistcall: newSearchResults: database is locked (5) (SQLITE_BUSY), select `dateTime` from `rdioScannerCalls` where true order by `dateTime` asc

Rebooted but that didn't clear it.

**Reply by AvalancheAddict** [reply]
I have seen the same problem twice now on new install of 6.1.3 on a Windows 2019 Server.  It showed a couple of pruning messages and nothing after that.  Files were building up, but nothing plays and no other log messages.  After a reboot it was ok again.
